loc_cts/tasks.py

```diff
@@ -5,7 +5,7 @@
 from celery.decorators import task
 from django.conf import settings
 
-from chronam.core import cts
+from chronam.loc_cts import cts
```

Comment:
Couldn't this also be `from . import cts` since it's in the same module?

Reply:
Yes. Will update. Thanks

Reply:
It could also be just `import cts` no?

Reply:
Currently, yes, but Python 3 doesn't have that implicit relative search path so this code would break then. `from . import cts` should work on both.